Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -896,6 +896,11 @@ def audit_revision_and_version_scheme
          current_revision == newest_committed_revision &&
          current_revision == previous_revision
         problem "'revision #{current_revision}' should be removed"
+      elsif current_version != newest_committed_version &&
```

Comment:
`(current_version != newest_committed_version || previous_version != newest_committed_version)` would allow this to be combined with the section above.